randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -1039,10 +1063,14 @@ Extra - room_id: [33]
   * L3 Hatch to Entrance Lobby/Door to Gerubus Gully
   * Extra - door_idx: (82,)
   > Pickup (Hidden Missile Tank)
-      Any of the following:
-          Can Use Power Bombs
-          Can Use Bombs and Can Use Springball
-          Morph Ball and Screw Attack and Knowledge (Beginner)
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Can Use Power Bombs
+              Screw Attack and Knowledge (Beginner)
+              Can Use Bombs and Can Use Springball
+              # https://youtu.be/T-MYy-tZVU0
```

Comment:
this accounts for not breaking the floor underneath you since you wont be able to reach the block nhj.
think on it more this could probably just be beginner